### ISSUE TITLE
TV3.se is missing in Allente and Tv.nu xmltv 

### ISSUE THREAD

**Issue by Jimpan81** [iptv-org/epg#1058]
### Source

Http://Tv.nu

### Notes

Swedish channel TV3.se is missing in Allente and Tv.nu xmltv , gone after update? Was working some days ago

**Comment by RevGear** [comment]
tv.nu and allente.se guides use the channel ID TV3Sweden.se, which doesn't exist in the database. I assume it's been deleted from the database as the most recent xml updates would have failed, although I skimmed through the most recent database updates and I can't where/if it was deleted.

tv.24.se guide uses the channel ID tv3.se, which does exist. Not sure if that's the same channel and the 2 xml files need updating or if TV3Sweden.se needs re-adding.

**Comment by Jimpan81** [comment]
Yes , tv3.se from tv24.se is the same channel as Tv3Sweden . From tv.nu and Allente
